sdk/sdk.go

```diff
@@ -55,6 +55,11 @@ func (sdk *SDK) GetExecutionStream(f *ExecutionFilter) *ExecutionListener {
 	return l
 }
 
+// UpdateExecution udpate execution that matches given hash.
+func (sdk *SDK) UpdateExecution(execHash, outputs []byte, reterr error) error {
+	return sdk.SubmitResult(execHash, outputs, reterr)
```

Comment:
not blocking

maybe better to move the content of `SubmitResult` here and to make it use `UpdateExecution` as we will remove `SubmitResult`

Reply:
Could be that way too, but I left this transformation when we're gonna remove the submit result api.